### PR TITLE
remove unused instance name

### PR DESCRIPTION
**Pull request by danfruehauf**
* use default.css (can be overriden by portalBranding)
 * use IMOS_logo.png (can be overriden by portalBranding)
 * eliminate use of INSTANCE_NAME and defaultInstanceName

Work for @pblain after he struggled a bit with a fair few bits of unnecessary configuration we still have in `Config.groovy`. Should still be improved more.

### CODE REVIEW

File: grails-app/conf/Config.groovy [hunk 1]
```diff
@@ -332,12 +323,7 @@ try {
     println "Loading external config from '$configurationPath'..."
 
     def startupConfig = new ConfigSlurper(grailsSettings.grailsEnv).parse(new File(configurationPath).toURI().toURL())
```
Comment (dnahodil) [inline grails-app/conf/Config.groovy]: I think this line can be removed as well

Comment (danfruehauf) [inline grails-app/conf/Config.groovy]: Yep, correct, no more references to that...

Comment (danfruehauf) [inline grails-app/conf/Config.groovy]: Removed.

**Comment by dnahodil** [comment]
This is nice, @danfruehauf. Good one.

**Pull request merged**